src/pages/ContestPage.tsx

```diff
@@ -56,26 +56,28 @@ const ContestPage = (props: Props) => {
       <ButtonBar>
         {nextContest ? (
           <LinkButton
+            id="next"
             key="next"
             primary={!!vote}
             to={`/contests/${nextContest && nextContest.id}`}
           >
             Next
           </LinkButton>
         ) : (
-          <LinkButton primary={!!vote} to="/review" key="review">
+          <LinkButton primary={!!vote} to="/review" key="review" id="next">
             Review
           </LinkButton>
         )}
         {prevContest ? (
           <LinkButton
+            id="previous"
             to={`/contests/${prevContest && prevContest.id}`}
             key="previous"
           >
             Previous
           </LinkButton>
         ) : (
-          <LinkButton key="backtostart" to="/start">
+          <LinkButton key="backtostart" to="/start" id="previous">
```

Comment:
🤔 Wondering if we should be concerned about these `id`'s being generic vs something more specific like `previous-contest`, `next-contest`, etc. No need to change now.

Reply:
I think `next` makes sense if it is truly the next action in the overall app, which I think is the case here.